index.html

```diff
@@ -0,0 +1,211 @@
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!--Header part starts-->
+    <header>
+      <nav class="navbar">
+        <img src="images/Logo Placeholder.png" alt="logo" />
+        <a class="toggler" target="_self" href="nav-mobile.html">
+          <img src="images/Icon - Menu.png" alt="icon image" />
+        </a>
+      </nav>
+    </header>
+    <!--Main part start-->
+    <section class="main">
+      <article class="main-article">
+        <h2>Hey There.
+            I’m Sazzad</h2>
+        <h3>I’am a Software Developer</h3>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+      </article>
+      <div class="social-media">
+        <ul class="social-media-icons">
+          <li>
+            <a href="https://github.com/SAZZAD1Q2" target="_blank" rel="noopener">
+                <img src="images/github.png" alt="">
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+                <img src="images/linkedin.png" alt="">
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+                <img src="images/finger.png" alt="">
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+                <img src="images/twitter.png" alt="">
+            </a>
+          </li>
+        
+          <li>
+            <a href="#" target="_blank">
+                <img src="images/M-img.png" alt="">
+            </a>
+          </li>
+        </ul>
+        <img class="main-img" src="images/main-img.png" alt="">
+      </div>
+    </section>
+    <section class="work">
+      <div class="work-header">
+        <h2 class="work-title">My Recent Works</h2>
+        <img src="images/work-title-img.svg" alt="">
+      </div>
+      <div class="work-body">
+       <img src="#" alt="">
+        <article class="work-article">
+           <h2>Multi-Post Stories</h2>
+           <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has text been the industry's standard dummy ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+           <ul class="work-links">
+            <li><a href="#">Html</a></li>
+            <li><a href="#">Css</a></li>
+            <li><a href="#">Javascript</a></li>
+            <li><a href="#">Ruby</a></li>
+           
+           </ul>
+           <button class="work-see-button">See Project</button>
+        </article>
+      </div>
+    </section>
+
+    <!--work card section start-->
+    <section class="work-cards">
+      <div class="card">
+        <h3>Profesional Art Printing Data</h3>
```

Comment:
I suggest you use the correct semantic tags for your work. You made every card into its own section which in this case renders your code spaghetti that is hard to read.
You should only have three sections for this:
    - `headline` -> which has your name, a description and social links
    - a works section that holds all your cards. THis cards can be `<articles>`
    - an about section - that has info about your about section